src/exercicio_nivelamento_2.ts

```diff
@@ -0,0 +1,239 @@
+let lista: { id: number; name: string; bio: string }[] = [
+  {
+    id: 1,
+    name: "Ada Lovelace",
+    bio: "Ada Lovelace, foi uma matemática e escritora inglesa reconhecida por ter escrito o primeiro algoritmo para ser processado por uma máquina",
+  },
+  {
+    id: 2,
+    name: "Alan Turing",
+    bio: "Alan Turing foi um matemático, cientista da computação, lógico, criptoanalista, filósofo e biólogo teórico britânico, ele é amplamente considerado o pai da ciência da computação teórica e da inteligência artificial",
+  },
+  {
+    id: 3,
+    name: "Nikola Tesla",
+    bio: "Nikola Tesla foi um inventor, engenheiro eletrotécnico e engenheiro mecânico sérvio, mais conhecido por suas contribuições ao projeto do moderno sistema de fornecimento de eletricidade em corrente alternada.",
+  },
+  {
+    id: 4,
+    name: "Nicolau Copérnico",
+    bio: "Nicolau Copérnico foi um astrônomo e matemático polonês que desenvolveu a teoria heliocêntrica do Sistema Solar.",
+  },
+];
+
+// a)Crie uma função que retorne a bio do id passado, versões normal, funcional e imperativo
+
+function getBioById(id: number) {
+  const index = lista.find((lista) => lista.id === id);
+  if (index) {
```

Comment:
se o find não encontrar o lista.id === id, ele não retornará -1?, se o index for -1 continuará caindo dentro desse if, creio que não foi esse o intuito da lógica